store.go

```diff
@@ -149,6 +151,11 @@ func (s *Store) PrimaryURL() string {
 	return s.primaryURL
 }
 
+// Candidate returns true if store is eligible to be the primary.
+func (s *Store) Candidate() bool {
+	return s.candidate
+}
+
```

Comment:
This previously had a lock around it but the field is only written once on initialization so it doesn't need one.